packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -178,13 +197,15 @@ const configure = ({
   onFlagsStateChange,
   onStatusStateChange,
   subscribeToFlagChanges = true,
+  throwOnInitializationFailure = false,
```

Comment:
I assume this means the default config lets the client fall back to the default flags if initialization fails. So the user decides if they want to throw the error (on initialization failure).

This would be a nice addition to the docs.

Reply:
Yes, that is correct. The default behaviour would be, as now, that the default flags kick-in or if none are specific everything defaulting to false. Given the user opts into the "new behaviour" with this option he/she can decide what to do. An React error boundary would catch it. One could also wrap flopflip itself in a custom (more local) error boundary and perform anything the `componentDidCatch`.

Reply:
Docs would be adequate and help. I would add them after we verified this works. Note also that the `subscribeToFlagChanges` is also a custom property (requested by another user of the library) who found it counter-intuitive that flags can update at any time. Which I still find bizarre cause it's the whole point :)

I would then document both options. Thanks for the reminder.